src/main/java/it/gov/pagopa/pu/workflow/controller/IngestionFlowControllerImpl.java

```diff
@@ -32,4 +36,15 @@ public ResponseEntity<WorkflowCreatedDTO> ingestPaymentsReportingFile(@PathVaria
         return ResponseEntity.status(201).body(response);
     }
 
+    @Override
+    public ResponseEntity<WorkflowCreatedDTO> ingestTreasuryOpi(@PathVariable("ingestionFlowFileId") Long ingestionFlowFileId) {
```

Comment:
This is not required, it will inherit it from the interface
```suggestion
    public ResponseEntity<WorkflowCreatedDTO> ingestTreasuryOpi(Long ingestionFlowFileId) {
```